pyproject.toml

```diff
@@ -49,13 +49,11 @@ classifiers = [
 keywords = ["evaluation", "nlp", "llm"]
 dependencies = [
     # Base dependencies
-    "transformers>=4.38.0",
-    "huggingface_hub==0.20.3",
+    "transformers[sklearn,sacrebleu]>=4.38.0",
```

Comment:
FYI @clefourrier I tweaked the deps here as well to align with `transformers`. Note that I only picked deps that were unpinned in both libs